src/server/system_services/schemas/bucket_schema.js

```diff
@@ -269,6 +269,22 @@ module.exports = {
                 reads: { type: 'integer' },
                 writes: { type: 'integer' },
             }
-        }
+        },
+        // Do we need enable/disable option? if yes, how to do that?
+        logging: {
+            type: 'object',
+            required: ['source_bucket', 'log_bucket', 'log_prefix'],
+            properties: {
+                source_bucket: {
```

Comment:
I think that `source_bucket` is redundant. This is a schema for an individual bucket, which is the source bucket